torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1633,6 +1633,24 @@ void InitXlaModuleBindings(py::module m) {
           }
           return result;
         });
+  // Load a list of local shards into an explicitly-sharded tensor. A shard must
+  // be provided for each device.
+  m.def("_load_local_shards", [](const at::Tensor& tensor,
+                                 std::vector<at::Tensor>& shards,
+                                 std::vector<std::string>& devices) {
+    XLATensorPtr xtensor = bridge::GetXlaTensor(tensor);
+    XLA_CHECK(xtensor->sharding_spec() != nullptr) << "Tensor is not sharded";
```

Comment:
make it more verbal, somthing along the line of `Can not load local shards into a non sharded tensor`